codecards/src/App.css

```diff
@@ -36,3 +48,130 @@
     transform: rotate(360deg);
   }
 }
+
+
+.header {
+  width: 100%;
+  background-color: #0c4768;
+  color: #eee;
+  height: 120px;
+  top: 0;
+  /* To stop content from overflowing on to the header */
+  z-index: 1000;
+}
+
+.header-elements {
```

Comment:
Really css related to certain components (e.g the header component) should be in a specific css file for that component (`Header.css`). This is one of the really powerful things about React is that we can modularise our code and keep code relevant to certain areas together - meaning that it's easy to make changes to specific areas.